app/main.py

```diff
@@ -1,3 +1,75 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> "Distance":
```

Comment:
The `__add__` method signature suggests that `other` should be an `int`, but the logic inside checks if `other` is a `Distance` instance. This is inconsistent and can lead to errors. Consider updating the method signature to `other: Union[Distance, int]` if you intend to support both types.